firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -24136,6 +26617,7 @@
 			baseConfigurationReference = 2CB288D02B07C6F300A8FCB3 /* EcosiaBeta.xcconfig */;
 			buildSettings = {
 				CODE_SIGN_ENTITLEMENTS = Ecosia/Entitlements/EcosiaBeta.entitlements;
+				DEVELOPMENT_TEAM = 33YMRSYD2L;
```

Comment:
There are still some other occurrences of this 🙈

Reply:
Yeah I saw them. I thought you wanted to remove the one for the Release only. Will remove all of them 💪